requirements.dev.txt

```
@@ -1,3 +1,5 @@
 inmanta-dev-dependencies[module]==1.76.0; python_version <= '3.6'
 inmanta-dev-dependencies[module]==2.95.0; python_version > '3.6'
 wheel==0.42.0
+# Pin to iso 6
+inmanta-core~=8.0,<8.8.dev
```

> is this correct wrt to the .dev in the constraint?

If the goal is to have it install dev even without `--pre`, then you need the `.dev` on both.
```suggestion
inmanta-core~=8.0.dev,<8.8.dev
```